### ISSUE TITLE
T1 L5 S11 Quiz question

### ISSUE THREAD

**Issue by sangxia** [udacity/sdc-issue-reports#582]
The question is "How might gradient descent fail to find the absolute minimum difference between the neural network's output and the known output?" The answer indicates that "There may be more than one absolute minima" is a correct option, and the explanation seems to indicate that if there are two absolute minima, then the function is not convex and optimizing over it is hard. I think this is not well phrased for two reasons:

1) having two absolute minima does not imply that the function is convex. consider the function f(x), where f(x)=-x for x<0, f(x)=0 for x in [0,1], and f(x)=x-1 for x>1. This function has infinitely many absolute minima and is still convex.

2) function being non-convex doesn't mean gradient there will be local minima that is not global minima. I think the wording could be improved here.

**Comment by awbrown90** [comment]
hi @sangxia thanks for the feedback. Yeah the wording here might be a little tricky since it says "How might" instead of "How". So yes there could be a function with two absolute minima where it can use gradient descent to successfully find the one of the absolute minimum but most likely such a function will not be the easy case.
